Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -39,6 +49,276 @@
 
 namespace DiscIO
 {
+std::once_flag RedumpVerifier::m_gc_download_flag;
+std::once_flag RedumpVerifier::m_wii_download_flag;
+RedumpVerifier::DownloadStatus RedumpVerifier::m_gc_download_status;
+RedumpVerifier::DownloadStatus RedumpVerifier::m_wii_download_status;
+
+void RedumpVerifier::Start(const Volume& volume)
+{
+  // We use GetGameTDBID instead of GetGameID so that Datel discs will be represented by an empty
+  // string, which matches Redump not having any serials for Datel discs.
+  m_game_id = volume.GetGameTDBID();
```

Comment:
Does it even make sense to trigger a download etc. when the game id is already empty?
Or do we just yolo it since Datel discs are really rare enough to be barely worth mentioning in code complexity?

Reply:
This is worth doing, because if the user has a good dump that is in redump then we will be able to confirm that it is a good dump. We just won't be able to know whether dumps that don't match a redump dump exactly are bad or not.

Reply:
How does Redump know if it doesn't have a game id?

Reply:
I'm not quite sure what you mean by that, but Redump's serial field is empty for all Datel discs (since it actually is based on the serial printed on the disc label rather than any data stored on the disc and since Datel discs don't have any serial printed on the disc label).

Reply:
```
(12:28:32) Bh44L: JosJuice: if the datel disc has no game id, what are you using to find the matching candidate in the redump file?
(12:28:38) JosJuice: Just the hashes
(12:28:46) Bh44L: oh
(12:29:02) Bh44L: so its basically a reverse brute-force, gotcha
(12:30:01) Bh44L: i guess you can mark those two as resolved then
(12:30:12) JosJuice: Okay
```